### PR TITLE
Add my last mod

### PR DESCRIPTION
**Pull request by Olympic1**
I just updated my Olympic1ARPIcons

### CODE REVIEW

**Comment by Olympic1** [comment]
Those checks find mistakes I can't see. I use "v1.4" because of the "find" command inside the install stanza, and placed comma's on the correct places. It also succesfully created a .ckan file.

File: NetKAN/Olympic1ARPIcons.netkan [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "abstract"       : "Add extra ARP icons for popular mods!",
+    "license"        : "CC-BY-NC-SA-4.0",
+    "author"         : "Olympic1",
+    "ksp_version"    : "1.0.4",
+    "resources" :
+    {
+        "homepage"    : "http://forum.kerbalspaceprogram.com/threads/102980",
+        "license"     : "https://github.com/Olympic1/Icon_Packs_KSP/blob/master/LICENSE_Olympic1ARPIcons.txt",
+        "repository"  : "https://github.com/Olympic1/Icon_Packs_KSP",
+        "kerbalstuff" : "https://kerbalstuff.com/mod/1095",
```
Comment (techman83) [inline NetKAN/Olympic1ARPIcons.netkan]: Extraneous comma here.

**Comment by techman83** [comment]
netkan.exe is a little more tolerant than the json spec. Extra commas are a pain!

**Comment by Olympic1** [comment]
Fixed. Commas are easy to look over at

**Comment by Olympic1** [comment]
Also a question: If I release a new update on github, will CKAN then automatically update the file or do I need to do it manually

File: NetKAN/Olympic1ARPIcons.netkan [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        "homepage"    : "http://forum.kerbalspaceprogram.com/threads/102980",
+        "license"     : "https://github.com/Olympic1/Icon_Packs_KSP/blob/master/LICENSE_Olympic1ARPIcons.txt",
+        "repository"  : "https://github.com/Olympic1/Icon_Packs_KSP",
+        "kerbalstuff" : "https://kerbalstuff.com/mod/1095"
+    },
+    "install" :
+    [
+        {
+            "find"       : "Olympic1ARPIcons",
+            "install_to" : "GameData"
```
Comment (plague006) [inline NetKAN/Olympic1ARPIcons.netkan]: Needs `"filter": "thumbs.db"`

**Comment by plague006** [comment]
Yep, new releases will be indexed automatically. The version numbers will be taken directly from GitHub and/or the included .version file. The only thing you'll have to change here in the .netkan file is when your mod is compatible with a new version of KSP.

**Comment by plague006** [comment]
Actually.. re:  #2144 if you're going to keep your KerbalStuff listing up to date it's better for us to scrape there than GitHub. We can get more information from the page itself (like the KSP version compatibility).

**Comment by ArnePeirs** [comment]
Wouldn't it be better that you make your `suggested` mods `recommended`, because it requires at least one to actually make sense to downoad

**Comment by Olympic1** [comment]
No, because if I should use `recommended`, CKAN will by default install the mods unless unchecked.

**Comment by ArnePeirs** [comment]
Jenkins approves :wink:

**Comment by plague006** [comment]
@Olympic1 will you be keeping the KerbalStuff page up to date?

**Comment by Olympic1** [comment]
yes

**Comment by plague006** [comment]
Passes local and Jenkins. Merging with my thanks. As someone who uses Alternate Resource Panel I'm looking forward to checking these out. =)

**Pull request merged**